<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionCountDialog</class>
 <widget class="QDialog" name="ConnectionCountDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Synapse Connections Count</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../rsc/espina.qrc">
    <normaloff>:/espina/espina.svg</normaloff>:/espina/espina.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,1,0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,1,0">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Connection criteria:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_criteriaLabel">
       <property name="text">
        <string>No criteria established</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_changeButton">
       <property name="text">
        <string>Change...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Total number of synapses:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_totalLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="m_invalidLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,1">
         <item>
          <widget class="QLabel" name="m_invalidTextLabel">
           <property name="text">
            <string>Invalid:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="m_invalidLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="m_invalidList"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Valid:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="m_fullLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="m_fullList"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
         <item>
          <widget class="QLabel" name="m_halfConnected">
           <property name="text">
            <string>Incomplete:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="m_halfLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="m_halfList"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
         <item>
          <widget class="QLabel" name="m_noConnected">
           <property name="text">
            <string>Unconnected:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="m_noneLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="m_noneList"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Double-click on a synapse to focus the view on it.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../rsc/espina.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConnectionCountDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
